PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,68 @@
+# Model Name
+
+## Use-cases
+Model use cases
+
+## Description
+Description of model
+
+## Model
+
+ |Model        |Download  | Download (with sample test data)|ONNX version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|Model Name       |ONNX Model download link with size  |  tar file containing ONNX model and synthetic test data|ONNX version used for conversion|Accuracy values |
+
+All ONNX models must pass the ONNX model checker before contribution. The snippet of code below can be used to perform the check. If any errors are encountered, it implies the check has failed.
+
+```
+import onnx
+from onnx import checker
+model_proto = onnx.load("path to .onnx file")
+checker.check_graph(model_proto.graph)
+```
+
+<hr>
+
+## Inference
+Step by step instructions on how to use the pretrained model and link to an example notebook/code. This section should ideally contain:
+
+### Input to model
+Input to network (Example: 224x224 pixels in RGB)
+
+### Preprocessing steps
+Preprocessing required
+
+### Output of model
+Output of network
+
+### Postprocessing steps
+Post processing and meaning of output
+<hr>
+
+## Dataset (Train and validation)
+This section should discuss datasets and any preparation steps if required.
+<hr>
+
+## Validation accuracy
+Details of experiments leading to accuracy and comparison with the reference paper.
+<hr>
+
+## Training
+Training details (preprocessing, hyperparameters, resources and environment) along with link to a training notebook (optional)
```

Comment:
Should also clarify that if model was not trained from scratch but instead sourced from elsewhere (e.g. converted from a CoreML model), the contributor should also explain what the process/source was for obtaining the final ONNX model

Reply:
done!